src/storage/txn/scheduler.rs

```diff
@@ -1597,10 +1597,14 @@ impl<E: Engine, L: LockManager> TxnScheduler<E, L> {
                 }
             }
         }
+
         // If it's not finished while the channel is closed, it means the write
         // is undeterministic. in this case, we don't know whether the
         // request is finished or not, so we should not release latch as
         // it may break correctness.
+        // However, not release latch will cause deadlock which may ultimately block all
+        // following txns, so we panic here.
+        panic!("response channel is unexpectedly dropped, cid {}", cid);
```

Comment:
loop transaction team for review this line.

Reply:
@MyonKeminta would you mind helping for revewing this?

Reply:
I've noticed the problem last week. I'm not sure if there are any corner cases that the channel is dropped *incorrectly* in raftstore, but this panic looks quite reasonable to me.

Reply:
By the way consider also printing `tag` here.

Reply:
> By the way consider also printing `tag` here.

Done